config/config_helpers.go

```diff
@@ -10,7 +11,8 @@ import (
 )
 
 var INTERPOLATION_SYNTAX_REGEX = regexp.MustCompile("\\$\\{.*?\\}")
-var HELPER_FUNCTION_SYNTAX_REGEX = regexp.MustCompile("\\$\\{(.*?)\\(\\)\\}")
+var HELPER_FUNCTION_SYNTAX_REGEX = regexp.MustCompile(`\$\{(.*?)\((.*?)\)\}`)
+var HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX = regexp.MustCompile(`(?P<ENV>[a-zA-Z_]+[a-zA-Z0-9_]+?)\,\s*\'(?P<default>.*?)\'`)
```

Comment:
If both the env var name and default value are quoted, the regex becomes a bit simpler. Nice use of named capture groups, BTW. I didn't realize Go supported those.